.gitignore

```diff
@@ -22,3 +22,6 @@ cabal.project.local~
 .HTF/
 .ghc.environment.*
 .ghci
+
+### Nix ###
+/result*
```

Comment:
why the `/` instead of all result* in any dir?

Reply:
Fixed